NumbersLibrary/RationalNumber.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+
+namespace NumbersLibrary
+{
+    //Дробные числа или Рациональные числа
+    public class RationalNumber
+    {
+        private int _Numerator;//числитель
+        private int _Denumerator;//знаменатель
+
+        public int Numerator { get { return _Numerator; }private set { _Numerator = value; } }
+        public int Denumerator { get { return _Denumerator; } private set { _Denumerator = value; } }
+
+
+        public RationalNumber (int Numerator,int Denumerator) 
+        {
+            if (Denumerator == 0) throw new ArgumentException("Знаменатель должен быть больше нуля");
+
+            _Numerator = Numerator;
+            _Denumerator = Denumerator;
+            ReduceToOptimal();
+        }
+
+        public RationalNumber(double Numerator, double Denumerator)
+        {
+            if (Denumerator == 0) throw new ArgumentException("Знаменатель должен быть больше нуля");
+
+            _Numerator = Convert.ToInt32( Numerator);
+            _Denumerator =Convert.ToInt32(Denumerator);
```

Comment:
```suggestion
            _Numerator = (int)Numerator;
            _Denumerator = (int)Denumerator;
```
А зачем вообще было придумано передавать вещественные числа???